package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Any specific reason why adding a `package.json` for Rotten Scripts?
`package.json` file is created when doing npm init or similar operations. Rotten Scripts isn't a JS npm package or a node app, thus please consider removing this file, it is not required.

Reply:
Ok, I will make the necessary changes.